pkgs/development/python-modules/typeid-python/default.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  poetry-core,
+  uuid6,
+  pytestCheckHook,
+  requests,
+  pyyaml,
+}:
+
+buildPythonPackage rec {
+  pname = "typeid-python";
+  version = "0.3.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "akhundMurad";
+    repo = "typeid-python";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-L1t2jcUWW/vCfSPHvxZgBBguNvrtWexCRAphoQBnAFM=";
+  };
+
+  pythonRelaxDeps = [
+    # For some reason, the uuid6 shipped in nixpkgs registers as version 0.0.0.dev0, so relax the dep
+    # since it actually is a valid version.
+    "uuid6"
+  ];
```

Comment:
https://github.com/NixOS/nixpkgs/pull/354857

Reply:
Will update when that PR is merged!